cross-chain/arbitrum/deploy_l2/02_deploy_arbitrum_wormhole_gateway.ts

```diff
@@ -0,0 +1,65 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers, deployments } = hre
+  const { log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // These are the fake random addresses for local development purposes only.
+  const fakeTokenBridge = "0x0af5DC16568EFF2d480a43A77E6C409e497FcFb9"
+  const fakeWormholeTBTC = "0xe1F0b28a3518cCeC430d0d86Ea1725e6256b0296"
+
+  const L2TokenBridge = await deployments.getOrNull("TokenBridge") // L2
+  const WormholeTBTC = await deployments.getOrNull("WormholeTBTC") // L2
```

Comment:
Thoughts on prefixing all the L2 contacts deployments with `Arbitrum` or `L2`?
```suggestion
  const ArbitrumTokenBridge = await deployments.getOrNull("ArbitrumTokenBridge") // L2
  const ArbitrumWormholeTBTC = await deployments.getOrNull("ArbitrumWormholeTBTC") // L2
```

@dimpar @pdyraga ^^

Reply:
a40b23e95094b721039947bace52d63308e3da49